glsl/rules.bzl

```diff
@@ -51,7 +51,7 @@ def _glsl_shader(ctx):
 
     shader = ctx.file.shader
 
-    spirv_name = shader.basename[:-len(shader.extension)] + "spv"
+    spirv_name = shader.basename[:-len(shader.extension)] + shader.extension + ".spv"
```

Comment:
I think this line is equivalent to

```python
spirv_name = shader.basename + ".spv"
```

See https://docs.bazel.build/versions/main/skylark/lib/File.html#basename

Reply:
You're right. I've pushed an update.